elbs/tf/all_elbs.tf

```diff
@@ -73,6 +74,7 @@ module "snippets" {
 }
```

Comment:
Similar to careers can you please add
`
health_check_http_path = "/healthz/"
`
now that snippets has a healthcheck endpoint?